app/components/RadioPlayer.vue

```diff
@@ -453,7 +453,8 @@ export default {
         }
         // Vue.prototype.$eventHub.$emit('np_updated', npNew);
       }).catch((error) => {
-        console.log('Network interrupted stream. Automatically reconnecting shortly...')
+        console.log('Network interrupted stream. Automatically reconnecting shortly...', error)
```

Comment:
If we log everything to sentry do we have reason to keep these console.logs?

Is the point not to remove all of them and replace them with sentry logging?

Reply:
I don't know, I think it is good for debugging, but to end-users it is not relevant at all